client-js/app/elements/labrad-plot.ts

```diff
@@ -181,6 +188,32 @@ export class Plot extends polymer.Base {
     }
   }
 
+  @observe('drawMode2D')
+  drawMode2DChanged(newMode: string, oldMode: string) {
+    switch (newMode) {
+      case 'dots':
+        this.$.dots.style.color = 'black';
+        this.$.rectfill.style.color = '#AAAAAA';
```

Comment:
Can we make the chart background white? I think the grey erodes a lot of the contrast. (I realize this may be leftover from my original D3 thing, but that was just copied from some example)

I think white background with grey gridlines, like what shows up on a google stock chart is pretty nice.